keps/sig-apps/2879-ready-pods-job-status/README.md

```diff
@@ -346,6 +348,17 @@ Pod created.
   to accept connections. On the other hand, the `Ready` condition is
   configurable through a readiness probe. If the Pod doesn't have a readiness
   probe configured, the `Ready` condition is equivalent to the `Running` phase.
-  
-  In other words, `Job.status.active` provides as the same behavior as
+
+  In other words, `Job.status.ready` provides as the same behavior as
   `Job.status.running` with the advantage of it being configurable.
+
+- We considered explore different batch periods for regular pod updates versus
```

Comment:
This section reads a bit weird.  I can't figure out the right way to phrase it though..

So you are saying:

We considered testing different batch periods but decided that since there was no bugs or requests raised, we are going forward with the defaults?

Reply:
Yes, I suggest to keep it simple. Proving that some configuration of delays is better than another in general will be very hard, I think results will depend on various factors. I think configurable delays might be something to consider in the future as a dedicated KEP, if needed.

Reply:
Also, it seems to me that changing the delays would help the KEP in beta for another release to test how it works after the change. wdyt @alculquicondor @soltysh ?